templates/S3/synapse-external-bucket.j2

```diff
@@ -128,6 +129,7 @@ Resources:
             Action:
               - "s3:ListBucket*"
               - "s3:GetBucketLocation"
+              - "s3:ListBucketMultipartUploads"
```

Comment:
This is redundant.  If you grant ListBucket* then you also grant ListBucketMultipartUploads